src/com/form-field.js

```diff
@@ -1,31 +1,26 @@
 /**
+ * Form Field
  * @author Deux Huit Huit
- *
- *  Form Field
- *
- *  Moment file : https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.15.0/moment.min.js
+ * @requires https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.15.0/moment.min.js
  */
-(function ($, w, doc, moment, undefined) {
+(function ($, win, undefined) {
```

Comment:
The global scope should be `global`, not `win` (win normally holds a jQueryfied version of window)

Reply:
true.

Reply:
fixed.